schema/input_fields/partner_categorytype.js

```diff
@@ -0,0 +1,15 @@
+import { GraphQLEnumType } from 'graphql';
+
+export default {
+  type: new GraphQLEnumType({
+    name: 'PartnerCategorytype',
```

Comment:
I lowercased this to indicate that it's not actually a `Type` but just something whose name includes the word "type". `PartnerCategoryType` is already a `Type` that represents a `PartnerCategory.` lmk if you think of a clearer way of naming this.